### PR TITLE
[hi] Localize glossary entry for “workload”

### PR DESCRIPTION
**Pull request by surbhiahuja**

### CODE REVIEW

File: content/hi/docs/reference/glossary/workload.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: कार्यभार(Workload)
+id: कार्यभार(Workloads)
```
Comment (sftim) [inline content/hi/docs/reference/glossary/workload.md:3]: This `id` is wrong. The `id` should be like this (same as the English original):
```suggestion
id: workloads
```

File: content/hi/docs/reference/glossary/workload.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+aka: 
+tags:
+- fundamental
+---
+   वर्कलोड कुबेरनेट्स पर चलने वाला एक एप्लिकेशन है।
+
+<!--more--> 
+
+विभिन्न मुख्य वस्तुएं जो विभिन्न प्रकार या कार्यभार के भागों का प्रतिनिधित्व करती हैं
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/workload.md:17]: I think this way the sentence is easier to understand. WDYT, thoughts @surbhiahuja?
```suggestion
वर्कलोड का विभिन्न प्रकार या कुछ खंड का प्रतिनिधित्व करने वाली विभिन्न कोर ऑब्जेक्ट में
```

File: content/hi/docs/reference/glossary/workload.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+डेमॉनसेट, डिप्लॉयमेंट, जॉब, रेप्लिकासेट और स्टेटफुलसेट ऑब्जेक्ट शामिल करें।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/workload.md:18]:
```suggestion
डेमॉनसेट, डिप्लॉयमेंट, जॉब, रेप्लिकासेट और स्टेटफुलसेट ऑब्जेक्ट शामिल हैं।
```

File: content/hi/docs/reference/glossary/workload.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+
+उदाहरण के लिए, एक वेब सर्वर और एक डेटाबेस वाला कार्यभार चल सकता है
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/workload.md:21]:
```suggestion
उदाहरण के लिए, एक वर्कलोड जिसमें एक वेब सर्वर और एक डेटाबेस हैं
```

File: content/hi/docs/reference/glossary/workload.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+एक {{<glossary_tooltip term_id="StatefulSet" >}} और वेब सर्वर में डेटाबेस
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/workload.md:22]:
```suggestion
डेटाबेस को {{<glossary_tooltip text="स्टेटफुलसेट" term_id="StatefulSet" >}} और
```

File: content/hi/docs/reference/glossary/workload.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+एक {{<glossary_tooltip term_id="Deployment" >}} में।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/workload.md:23]:
```suggestion
वेब सर्वर को {{<glossary_tooltip text="डिप्लॉमेंट" term_id="Deployment" >}} में चला सकते हैं।
```

**Review by Babapool — COMMENTED** [review]
Hope you find these suggestions useful. Thanks:

**Comment by Babapool** [comment]
/retitle [hi] Localize glossary entry for “workload”

File: content/hi/docs/reference/glossary/workload.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+---
+title: Workload
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/workload.md:2]: Title needs to be translated too.

File: content/hi/docs/reference/glossary/workload.md (hunk 5 above)
Comment (divya-mohan0209) [reply content/hi/docs/reference/glossary/workload.md:22]: Agree with  suggestion made by @Babapool

File: content/hi/docs/reference/glossary/workload.md (hunk 6 above)
Comment (divya-mohan0209) [reply content/hi/docs/reference/glossary/workload.md:23]: Agree with suggestion made by @Babapool

File: content/hi/docs/reference/glossary/workload.md (hunk 2 above)
Comment (divya-mohan0209) [reply content/hi/docs/reference/glossary/workload.md:17]: I still don't think the two sentences together actually make sense.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/workload.md (hunk 7 above)
Comment (divya-mohan0209) [reply content/hi/docs/reference/glossary/workload.md:2]: Please address this.

File: content/hi/docs/reference/glossary/workload.md (hunk 2 above)
Comment (divya-mohan0209) [reply content/hi/docs/reference/glossary/workload.md:17]: Please could we rewrite this @surbhiahuja ?

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/workload.md [hunk 8]
```diff
@@ -14,8 +14,7 @@ tags:
 
 <!--more--> 
 
-विभिन्न मुख्य वस्तुएं जो विभिन्न प्रकार या कार्यभार के भागों का प्रतिनिधित्व करती हैं
-डेमॉनसेट, डिप्लॉयमेंट, जॉब, रेप्लिकासेट और स्टेटफुलसेट ऑब्जेक्ट शामिल हैं।
+विभिन्न मुख्य वस्तुएं जो विभिन्न प्रकार के कार्यभार का प्रतिनिधित्व करती हैं जिस में डेमॉनसेट, डिप्लॉयमेंट, जॉब, रेप्लिकासेट और स्टेटफुलसेट ऑब्जेक्ट शामिल हैं।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/workload.md:17]:
```suggestion
विभिन्न कोर ऑब्जेक्ट्स जैसे डेमनसेट, डिप्लॉयमेंट, जॉब, रेप्लिकासेट और स्टेटफुलसेट एक कार्यभार के विभिन्न प्रकारों या भागों को प्रतिनिधित्व करते हैं।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
डिप्लॉयमेंट spelling needs to be standardized across the docs. Please use the same spelling as used elsewhere in the docs. 

Also changed the grammar a little bit to simplify & clarify what is being conveyed.

**Comment by divya-mohan0209** [comment]
Hello @surbhiahuja , thank you for your efforts on this PR. However, it has been inactive for ~5 months & as a result, I shall be closing it off. Whenever you have the cycles/bandwidth to work on addressing the feedback, please consider reopening it.

/close

**Pull request closed without merging**